src/main/scala/catslib/Apply.scala

```diff
@@ -48,7 +46,7 @@ object ApplySection extends FlatSpec with Matchers with org.scalaexercises.defin
     * Since `Apply` extends `Functor`, we can use the `map` method from `Functor`:
     */
   def applyExtendsFunctor(res0: Option[String], res1: Option[Int], res2: Option[Int]) = {
-    import cats.std.all._
+    import cats.implicits._
```

Comment:
I think these should have the actual import of the feature used which is a lot more precise than importing all of cats implicits.